### PR TITLE
Additional end-to-end tests

### LINKED ISSUE #1209: Add web browser tests
Our current end-to-end tests only cover the chart installation and tests. In order to avoid major regressions we should include a basic coverage of web browser tests to ensure we don't break the current UX.

### PR DESCRIPTION
**Pull request by andresmgot**
Closes #1209 

Add two new tests:

 - Use a non-privileged user to create a release (that fails)
 - Create a new repository

See inline comments.

### CODE REVIEW

File: chart/kubeapps/templates/apprepository-rbac.yaml [hunk 1]
```diff
@@ -86,10 +86,7 @@ rules:
   resources:
   - apprepositories
   verbs:
-  - get
-  - create
-  - update
-  - delete
+  - "*"
```
Comment (andresmgot) [inline chart/kubeapps/templates/apprepository-rbac.yaml]: the `write` role was missing the `list` permissions

File: integration/use-cases/create-registry.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  await expect(page).toFillForm("form", {
+    token: process.env.ADMIN_TOKEN
+  });
+
+  await expect(page).toClick("button", { text: "Login" });
+
+  // Double click to show configuration
+  await expect(page).toClick("a", { text: "Configuration" });
+  await expect(page).toClick("a", { text: "Configuration" });
```
Comment (andresmgot) [inline integration/use-cases/create-registry.js]: I tried to `hover` over the link but it doesn't have any valid selector

File: chart/kubeapps/templates/apprepository-rbac.yaml (hunk 1 above)
Comment (absoludity) [reply chart/kubeapps/templates/apprepository-rbac.yaml]: So using "*" also adds `patch` and `watch`. Not sure if that's intended.

Comment (andresmgot) [reply chart/kubeapps/templates/apprepository-rbac.yaml]: yes, I mean, I would expect this role to be "admin" for apprepositories

**Pull request merged**